### PR TITLE
blog: upload chaos mesh x skywalking

### PR DESCRIPTION
**Pull request by milasuperstar**
This is to upload the blog "Chaos Mesh + SkyWalking: Better Observability for Chaos Engineering".

Signed-off-by: Mila Wu <wumila1996@gmail.com>

### CODE REVIEW

File: blog/2021-12-16-chaos-mesh-skywalking.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+![SkyWalking Dashboard](/img/skywalking-dashboard.png)
+
+## Step 4: Inject failures via Chaos Mesh and observe results 
+
+After you finish the three steps above, you can use the Chaos Dashboard to simulate stress scenarios and observe the change in service performance during chaos experiments. 
+
+![StressChaos on Chaos Dashboard](/img/chaos-dashboard-stresschaos.png)
+
+The following sections describe how service performance varies under the stress of three chaos conditions.
```
Comment (dcalvin) [inline blog/2021-12-16-chaos-mesh-skywalking.md:75]: Should use bullet list for the different scenatios instead of H3 heading.

Comment (dcalvin) [inline blog/2021-12-16-chaos-mesh-skywalking.md:75]:
```suggestion
The following sections describe how service performance varies under the stress of three chaos conditions:
```

**Review by dcalvin — APPROVED** [review]
LGTM

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**